.github/workflows/golangci-lint.yml

```
@@ -27,13 +27,15 @@ jobs:
           export PATH=$PATH:/home/runner/go/bin/
 
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v3.7.0
+        uses: golangci/golangci-lint-action@v4.0.0
         with:
           # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
-          version: v1.56
+          version: v1.57.2
```

nit: we could allow patch updates
```suggestion
          version: v1.57
```